src/host/cookedRead.hpp

```diff
@@ -112,7 +112,7 @@ class CookedRead final : public ReadData
     COORD _promptStartLocation;
     // the location of the cursor before a popup is launched
     COORD _beforePopupCursorPosition;
-    CommandHistory* _pCommandHistory; // non-ownership pointer
+    std::shared_ptr<CommandHistory> _pCommandHistory; // CommandHistory pointer

```

Comment:
I'd just remove the comment on this line totally, it serves no purpose anymore.

Reply:
done